webcompat/views.py

```diff
@@ -207,7 +209,13 @@ def show_issue(number):
     if session.get('show_thanks'):
         flash(number, 'thanks')
         session.pop('show_thanks')
-    return render_template('issue.html', number=number)
+    content = render_template('issue.html', number=number)
+    response = make_response(content)
+    # We can cache for one year. Etag is based on the content.
+    response = set_cache_control(response, 31536000)
```

Comment:
Ah! I might be a bit too agressive on the caching here.  I was wondering why the browser was not sending any conditional requests to the server

`Cache-Control: private, max-age=31536000`

which sets it to **1 year**. From the browser point of view it takes priority on the etag, and the browser will not even attempt to do `if-none-match` (aka conditional request). As long as the resource is in the cache and the 1 year has not passed, no attempt of revalidation will be done.

So basically, we need to decide what is the right balance in between:
* cache-control: Keeping the resources for x seconds. No hit to the server.
* etag: new etag when the resource has changed. A hit to the server with an eventual empty body and a 304. Or the full 200, if it has changed.